src/modality-specific-files/motion.md

```diff
@@ -0,0 +1,205 @@
+To store the differences between recording onsets, [`scans.tsv`](../modality-agnostic-files.md#scans-file) files can be used.
+
+To store events which relate to a tracking system, it is recommended to use designated events files per tracking system.
+Such an events file name would include the `tracksys` entity and look like `sub-<label>[_ses-<label>]_task-<label>[_acq-<label>][_run-<index>]_tracksys-<label>_events.tsv`.
+The onsets in the event file can be related to the starting time of the tracking system in the `scans.tsv` file.
+
+### Sidecar JSON (`*_motion.json`)
+
+#### Task information
+
+{{ MACROS___make_sidecar_table("motion.motionTaskInformation") }}
+
+#### Hardware information
+
+{{ MACROS___make_sidecar_table("motion.motionHardware") }}
+
+#### Insitution information
+
+{{ MACROS___make_sidecar_table("motion.motionInstitutionInformation") }}
+
+#### Motion specific fiels
+
+Motion specific fields MUST be present:
+
+{{ MACROS___make_sidecar_table("motion.motionRequired") }}
+
+Motion specific fields SHOULD be present:
+
+{{ MACROS___make_sidecar_table("motion.motionRecommended") }}
+
+Restricted keyword list for field `RotationRule`:
+
+| **Keyword** | **Description**                                                                                                                                                                 |
+| ----------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| left-hand   | Rotation is following the left-hand convention, such that the left thumb points to the positive end of the spatial axis, and the fingers curl along the orientation rotation.   |
+| right-hand  | Rotation is following the right-hand convention, such that the right thumb points to the positive end of the spatial axis, and the fingers curl along the orientation rotation. |
+
+Restricted keyword list for field `RotationOrder`:
+
+| **Keyword** | **Description**                                                                         |
+| ----------- | --------------------------------------------------------------------------------------- |
+| XYZ         | Sequence in which elemental rotations are applied to define an orientation in 3D space. |
+| XZY         | Sequence in which elemental rotations are applied to define an orientation in 3D space. |
+| YXZ         | Sequence in which elemental rotations are applied to define an orientation in 3D space. |
+| YZX         | Sequence in which elemental rotations are applied to define an orientation in 3D space. |
+| ZXY         | Sequence in which elemental rotations are applied to define an orientation in 3D space. |
+| ZYX         | Sequence in which elemental rotations are applied to define an orientation in 3D space. |
```

Comment:
@JuliusWelzel @sjeung I think this entire table can be removed.

The `RotationOrder` field is rendered like this in the spec, which seems like enough information:

![grafik](https://user-images.githubusercontent.com/9084751/220084287-b3d7590e-c9d2-4640-8ecd-541c9bbd7c7a.png)

What do you think?